src/components/MintingInfo/MintingInfo.component.tsx

```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import { MintingInfoPros } from './MintingInfo.types';
+
+export const MintingInfo = ({ data }: MintingInfoPros) => (
+  <>
+    {data.map((d, i) => (
```

Comment:
What do you think about that? Just a suggestion
```suggestion
    {data.map(({ label, value, isProminant }, index) => (
```